README.md

```diff
@@ -6,26 +6,25 @@
 [![Codecov][codecov-src]][codecov-href]
 [![bundle][bundle-src]][bundle-href]
 
-
-
 **▶️ Check [online playground](https://untyped.unjs.io)**
 
+## Install
 
-## Usage
-
-### Install package
+```sh
+yarn add untyped
+```
 
-Install `untyped` npm package:
+```sh
+pnpm add untyped
+```
 
 ```sh
-yarn add untyped
-# or
-npm i untyped
+npm i --save untyped
```

Comment:
I believe `--save` is default behavior of npm in latest versions. Also do you mind to follow the [template](https://github.com/unjs/template#usage)? I also like new display but we need to update all unjs repos to apply this.

Reply:
My mistake, I was unaware of the template until now. I'll tidy things up now to be more consistent. Thanks for bringing me up to speed! Between `npm --save` and `a[name]` I guess I was looking a bit outdated 😅